### DISCUSSION TITLE
New preview release

### DISCUSSION THREAD

**Discussion by jp-weber** [CommunityToolkit/Windows#340] [General]
Is there a date when a new preview version will be published?

**Comment by michael-hawker** [comment]
Every commit to main goes to our devops feed here: https://dev.azure.com/dotnet/CommunityToolkit/_packaging?_a=feed&feed=CommunityToolkit-MainLatest

Believe @Arlodotexe was going to copy our old wiki's preview page to the new one too. https://aka.ms/wct/wiki/previewpackages

**Comment by Panda-Sharp** [comment]
Thanks @michael-hawker 

Any plan for an "official" Nuget preview? Or do you publish only final builds on Nuget?

**Reply by michael-hawker** [reply]
If we are gearing up for a release, we'll generally release a preview and/or release candidate a week or two before hand to get more exposure for final bug finding beforehand.

We just don't have any set plans on shipping an 8.1 yet. We just got done with updating to .NET 8 with the RID changes that happened, so that was the big hurdle we've been getting over. We're also trying to resolve an issue for migration of #241 with PRI files and it not working for UWP.

Once that's done, I'd like to get some focus on our docs and wiki first, as they need some love. But then we should have a couple of new things and enough updates that we could justify a release, there's just been a lot of moving pieces lately.
